trade-finance-manager-ui/server/api.get-uk-bank-holidays.test.js

```diff
@@ -0,0 +1,34 @@
+const axios = require('axios');
```

Comment:
This is just split from the `api.test.js` file as it was far too big. No changes.